src/App.tsx

```diff
@@ -1,14 +1,59 @@
-/* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useMemo, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { Todo } from './types/Todo';
+import { getTodos } from './api';
+
+const getPreparedTodos = (todos: Todo[], query: string, filterType: string) => {
+  let copyTodos = [...todos];
+
+  if (query) {
+    copyTodos = copyTodos.filter(todo => {
+      const normalizedTitle = todo.title.toLowerCase();
+
+      return normalizedTitle.toLowerCase().includes(query.toLowerCase());
```

Comment:
Query should be trimmed
![image](https://github.com/mate-academy/react_dynamic-list-of-todos/assets/54936044/6c9b4c83-9f24-47ef-a348-4d48f33a32ab)